apps/react/src/features/genres/components/GenresFilter/GenresFilter.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC, memo } from 'react';
+import { FILTER_QUERY_PARAMS } from '@js-camp/react/config';
+import { useQueryParams } from '@js-camp/react/hooks/useQueryParams';
+import { GenresTextField } from '@js-camp/react/features/genres/components/GenresTextField/GenresTextField';
+import { GenresQueryParams } from '@js-camp/react/models/genres-query-params';
+import { useAppSelector } from '@js-camp/react/store';
+import { selectAreGenresLoading } from '@js-camp/react/store/genre/selectors';
+
+const filters = [
+	{ value: '', label: 'Default' },
+	{ value: GenresQueryParams.FilterType.Genres, label: 'Genres' },
+	{ value: GenresQueryParams.FilterType.ExplicitGenres, label: 'Explicit genres' },
+	{ value: GenresQueryParams.FilterType.Themes, label: 'Themes' },
+	{ value: GenresQueryParams.FilterType.Demographics, label: 'Demographics' },
+];
```

Comment:
To help us avoid any possible mistypes (you may wanna adjust the naming if needed, it's just a basic suggestion to improve type safety):

```suggestion
type Filter = Readonly<{
	value: GenresQueryParams.FilterType;
	label: string ;
}>

const filters: Filter[]  = [
	{ value: '', label: 'Default' },
	{ value: GenresQueryParams.FilterType.Genres, label: 'Genres' },
	{ value: GenresQueryParams.FilterType.ExplicitGenres, label: 'Explicit genres' },
	{ value: GenresQueryParams.FilterType.Themes, label: 'Themes' },
	{ value: GenresQueryParams.FilterType.Demographics, label: 'Demographics' },
];
```

Reply:
Good suggestion bro, love it ❤️